deployment/ray/mednist_classifier_ray.ipynb

```diff
@@ -348,26 +348,12 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    
</pre><p>Here the imports need to be restored.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1618/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>